cmake/FindPETSc.cmake

```diff
@@ -350,6 +350,14 @@ int main(int argc,char *argv[]) {
   mark_as_advanced (PETSC_INCLUDES PETSC_LIBRARIES PETSC_COMPILER PETSC_DEFINITIONS PETSC_MPIEXEC PETSC_EXECUTABLE_RUNS)
 endif ()
 
+if (NOT PETSC_INCLUDES)
+  include(FindPkgConfig)
+  pkg_check_module(PkgPETSC PETSc>3.4.0 petsc>3.4.0)
```

Comment:
Should be `pkg_search_module` to find first of list? `pkg_check_modules` needs all modules to be found

Reply:
Thanks for noticing, I thought only the missing s would be the different. Fixed now.